app/middleware/electron.js

```diff
@@ -44,7 +44,9 @@ export default (store: Store) => (next: (action: Action) => void) => async (
       // eslint-disable-next-line no-console
       console.error(`No view registered for ${name}!`);
     } else {
-      viewState.browserView.webContents.loadURL(targetUrl);
+      viewState.browserView.webContents.loadURL(targetUrl).then(() => {
```

Comment:
That doesn't work as I would have expected. But somewhere this cookie should be set

Reply:
could you move this into a separate PR please?